src/main/resources/interface-specification.yml

```diff
@@ -124,19 +103,23 @@ paths:
           schema:
             type: string
             maxLength: 32
+        - in: query
+          name: page
```

Comment:
same for `min` constraint + default value

Reply:
[Same](https://github.com/pagopa/interop-be-party-registry-proxy/pull/115#discussion_r960594340)